dist/src/main/assembly/conf/logback.xml

```diff
@@ -16,7 +16,9 @@
 -->
 <configuration>
     <property name="logging.basename" scope="context" value="brooklyn-clocker" />
-    <logger name="brooklyn.clocker" level="DEBUG" />
+
+    <logger name="clocker" level="DEBUG" />
```

Comment:
Yes, was wondering why I didn't get any logging after the package renaming, and realised I needed to add a new logger...